src/components/Node/Node.tsx

```diff
@@ -50,30 +56,42 @@ const Node: FC<NodeProps & { ref: Ref<HTMLDivElement> }> = forwardRef<HTMLDivEle
       top: yCoordinate,
       left: xCoordinate
     };
+
     const backgroundColorClass = isOnPatientPath ? styles.onPatientPath : styles.notOnPatientPath;
-    const currentNodeClass = isCurrentNode ? styles.current : '';
-    const expandedNodeClass = isCurrentNode
-      ? styles.childCurrent
-      : isOnPatientPath
-      ? styles.childOnPatientPath
-      : styles.childNotOnPatientPath;
+    const isActionable = isCurrentNode && !documentation;
+    const topLevelClasses = [styles.node, backgroundColorClass];
+    let expandedNodeClass = '';
+    if (expanded) topLevelClasses.push(nodeStyles.expanded);
+    if (isActionable) {
+      topLevelClasses.push(styles.actionable);
+      expandedNodeClass = styles.childActionable;
+    } else {
+      expandedNodeClass = isOnPatientPath
+        ? styles.childOnPatientPath
+        : styles.childNotOnPatientPath;
+    }
     const isGuidance = isGuidanceState(pathwayState);
+    // TODO: how do we determine whether a node has been accepted or declined?
+    // for now:
+    // if it's a non-actionable guidance state on the path: accepted == has documentation
+    // if it's actionable, not guidance or not on the path: null
+    const isAccepted =
+      isOnPatientPath && isGuidance && !isActionable ? documentation != null : null;
```

Comment:
it would be more readable to break this into two parts.

One for `const nodeState = isOnPatientPath && isGuidance && !isActionable` and another for `nodeState ? documentation !=null : null`

the `nodeState` name I put in is not great though.